src/main/app/utils/numberFormatter.ts

```diff
@@ -1,6 +1,6 @@
 import * as numeral from 'numeral'
 
-export const NUMBER_FORMAT = '$0,0.00'
+export const NUMBER_FORMAT = '$0,0[.]00'
```

Comment:
This makes the money formatter not display pennies if there are none, i.e. no `.00`.